_format/1.1/index.md

```diff
@@ -1781,6 +1822,183 @@ parameter from this specification, it **MUST** return `400 Bad Request`.
 
+## <a href="#profiles" id="profiles" class="headerlink"></a> Profiles
+
+JSON API supports the use of "profiles" as a way to indicate additional
+semantics that apply to a JSON:API request/document, without altering the 
+basic semantics described in this specification.
+
+[RFC 6906](https://tools.ietf.org/html/rfc6906) covers the nature of profile
+identification:
+
+> Profiles are identified by URI. However, as is the case with, for
+  example, XML namespace URIs, the URI in this case only serves as an
+  identifier, meaning that the presence of a specific URI has to be
+  sufficient for a client to assert that a resource representation
+  conforms to a profile. Thus, clients SHOULD treat profile URIs as
+  identifiers and not as links.
+
+However, to aid human understanding, profile URIs **SHOULD** return 
+documentation of the profile.
+
+A profile **MAY** assign meaning to particular elements of the document
+structure, such as resource attributes or members of meta objects, and even
+to query parameters whose behavior is left up to each implementation, such 
+as `filter` and all those that contain a non a-z character.
+
+The scope of a profile **MUST** be clearly delineated. The elements reserved by 
+a profile, and the meaning assigned to those elements, **MUST NOT** change over 
+time or else the profile **MUST** be considered a new profile with a new URI. 
+
+Profiles **MAY** be updated over time to add new capabilities. However, any such 
+changes **MUST** be [backwards and forwards compatible](http://www.w3.org/2001/tag/doc/versioning-compatibility-strategies#terminology),
+in order to not break existing users of the profile. This requirement usually
+limits changes to adding optional keys within objects specified in the profile's
+original definition.
+
+For example, let's say that a profile reserves a `timestamps` member in the
+`meta` object of every resource. Originally, this profile defines the value
+of `timestamps` as an object that must contain one member: `created`. The
+profile could evolve to allow an optional member, `updated`, in the `timestamps`
+object. But it could not make that member required, nor could it introduce
+a new sibling to `timestamps`.
+
+Profiles **SHOULD** reserve at least one object-valued member, and **SHOULD**
+consider reserving an object-valued member anywhere they expect to potentially 
+add new features over time.
+
+> Note: When a profile changes its URI, a huge amount of interoperability is lost.
+> Users that reference the new URI will have their messages not understood by
+> implementations still aware only of the old URI, and vice-versa. 
+> Accordingly, the advice above is aimed at allowing profifiles to grow
+> without needing to change their URI.
+
+### <a href="#profile-keywords" id="profile-keywords" class="headerlink"></a> Profile Keywords
+
+A profile **SHOULD** explicitly declare "keywords" for any elements that it
```

Comment:
*How* can a profile explicitly declare a keyword?

Reply:
The profile would declare the keyword in its definition. Part of the problem is that this PR never actually says what a profile *is*. We go straight from talking about what they do ("a way to indicate additional semantics") to how they're identified ("by URI"). If we import some language from #1195:

> A [profile] is a small, separate specification that defines...

(We have to update what's after the ...)

Then, it's clear that you declare keywords in that specification